src/lib.rs

```diff
@@ -73,8 +73,10 @@ mod tests;
 pub mod prelude {
     pub use crate::auto::traits::*;
     // See "Re-export dependencies above".
+    pub use gio;
     #[doc(hidden)]
     pub use gio::prelude::*;
+    pub use glib;
```

Comment:
I have a feeling these belong to the top-level but outside of `prelude`, i.e. the same way that there is a `::gio::glib` but no `::gio::prelude::glib`. Consumers can always pretty-rename them with `use ostree::gio as gio`.

Reply:
We already have them at the toplevel...so are you saying this PR is unnecessary or that it's wrong?

As I said in the commit message, the goal seems clear that if you import the prelude, you should get the toplevel dependencies directly too *in addition* to their preludes, right?

To be clear I'm testing with this https://github.com/cgwalters/ostree-rs-ext/commit/8b6fa7d4a0d544e101474c39b98ac6d265c93eb4

Reply:
The linked webpage has the example
```
use gtk::prelude::*;

// ...
let x = cairo::something();
```

And I actually verified with the current `gtk` crate that's true.

Reply:
I'm slightly out of clues, as I don't see the module neither in the [docs](https://docs.rs/gtk/0.14.0/gtk/prelude/index.html) (but may be hidden) nor in the [prelude source](https://github.com/gtk-rs/gtk3-rs/blob/0.14.0/gtk/src/prelude.rs) (which doesn't reference `cairo` at all).

Reply:
Right, I looked at the source too and also could not figure out how it works.

Reply:
I did a quick test with https://gist.github.com/lucab/d8e641f411b43264726304cb6ec62c71 and the compiler also seems to have trouble finding the module through the prelude.

Reply:
I've asked on IRC and apparently the blogpost was wrongly worded, that section was supposed to talk about traits only. It has been updated in that direction now.
I think we can follow the same approach for ostree-rs, i.e. keeping the current status without this PR. Your example will then be:
```
use ostree::gio;

let c = gio::Cancellable::new();
```

Reply:
Ah.  And I think I figured out the discrepancy between your testing and mine.  I was testing via an integration test inside the `gtk-rs-core/gio/tests` path, and I was able to directly use `glib` because it's part of the workspace.